glslc/src/file_compiler.h

```diff
@@ -64,12 +64,22 @@ class FileCompiler {
     output_file_name_ = file;
   }
 
-  // Sets the flag to indicate emitting SPIRV binary code as a C-style
-  // initializer list for a 32-bit unsigned integer array.
-  void SetEmitSpirvBinaryAsCInitializerList() {
-    emit_binary_as_c_init_list_ = true;
+  // Sets the flag to indicate emitting SPIR-V binary code directly.
+  void SetEmitSpirvBinaryAsBinary() {
```

Comment:
I recommend having a single setter that takes a SpirvBinaryEmissionFormat enum value as its parameter.